### PR TITLE
Pagination refactor for org users, teams & archives

### PR DESCRIPTION
**Pull request by future-pirate-king**

### CODE REVIEW

File: app/components/organization-member/list/index.js [hunk 1]
```diff
@@ -1,95 +1,165 @@
...
             textAlign: 'center',
           }
         : null,
     ].filter(Boolean);
-  }),
+  }
 
-  /* computed.sort compares the values with previous value,
-    if it's not change same value will return everytime */
+  // newMembersObserver = observer(
```
Comment (Yibaebi) [inline app/components/organization-member/list/index.js:73]: Comments?

File: app/components/organization-team/member-list/index.js [hunk 2]
```diff
@@ -1,70 +1,127 @@
 /* eslint-disable ember/no-classic-components, ember/no-mixins, ember/no-classic-classes, ember/require-tagless-components, ember/avoid-leaking-state-in-ember-objects, prettier/prettier, ember/no-get, ember/no-observers */
```
Comment (Yibaebi) [inline app/components/organization-team/member-list/index.js:1]: I think these comments can be removed.

**Review by Yibaebi — APPROVED** [review]

File: app/components/organization-member/list/index.js (hunk 1 above)
Comment (future-pirate-king) [reply app/components/organization-member/list/index.js]: removed

File: app/components/organization-team/member-list/index.js (hunk 2 above)
Comment (future-pirate-king) [reply app/components/organization-team/member-list/index.js]: removed

**Pull request merged**